post_office/admin.py

```diff
@@ -83,7 +83,8 @@ def requeue(modeladmin, request, queryset):
 
 @admin.register(Email)
 class EmailAdmin(admin.ModelAdmin):
-    list_display = ['id', 'to_display', 'shortened_subject', 'status', 'last_updated', 'scheduled_time', 'use_template']
+    list_display = ['truncated_message_id', 'to_display', 'shortened_subject', 'status', 'last_updated', 'scheduled_time', 'use_template']
+    readonly_fields = ['message_id']
```

Comment:
There are two `readonly_fields` definition, here and in line 88. They should be combined.

Reply:
Ups, missed that.
Anyway, the Message-ID now is only rendered if available.